ts/components/screens/BaseHeader.tsx

```diff
@@ -21,11 +20,27 @@ import GoBackButton from "../GoBackButton";
 import InstabugChatsComponent from "../InstabugChatsComponent";
 import SearchButton, { SearchType } from "../search/SearchButton";
 import AppHeader from "../ui/AppHeader";
+import I18n from "../../i18n";
+
+type HelpButtonProps = {
+  onShowHelp: () => void;
+};
+
+const HelpButton: FC<HelpButtonProps> = ({ onShowHelp }) => (
+  <ButtonDefaultOpacity
+    hasFullHitSlop
+    onPress={onShowHelp}
+    transparent={true}
+    accessibilityLabel={I18n.t(
+      "global.accessibility.contextualHelp.open.label"
+    )}
+    accessibilityHint={I18n.t("global.accessibility.contextualHelp.open.hint")}
+  >
+    <IconFont name={"io-question"} />
+  </ButtonDefaultOpacity>
+);
 
 const styles = StyleSheet.create({
-  helpButton: {
```

Comment:
This is what could happen when you do, refactoring. Something could be missed

#### `top: master` - `bottom: now`
<img src="https://user-images.githubusercontent.com/822471/105152232-3199e500-5b07-11eb-82dc-b27298f98a87.png" height="650"/>